src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -706,6 +706,8 @@ protected void addTranslations()
 		addTooltip(MSItems.GOLDEN_SPORK, "A spoon and fork with all the awfulness of gold tools? Sign me up!");
 		add(MSItems.BIDENT, "Bident");
 		addTooltip(MSItems.BIDENT, "Double trouble for all your bubble scuffle needs");
+		add(MSItems.EDISONS_FURY, "Edison's Fury");
+		add(MSItems.EDISONS_SERENITY, "Edison's Fury");
```

Comment:
This is dependent on preference, but it would be smart to match the tooltip name to the registry name if you want the serenity/fury naming scheme to be identifiable by players and be understood in the code